utility/utils.py

```diff
@@ -725,20 +724,24 @@ def destroy_cluster(cluster_path):
         return StatusOfTest.FAILED
 
 
-def get_openshift_installer(version=defaults.INSTALLER_VERSION):
+def get_openshift_installer(
+    version=DEPLOYMENT['installer_version'],
+    bin_dir=RUN['bin_dir'],
```

Comment:
this should also be DEPLOYMENT var ?

Reply:
IMHO, I would put all three variables: `installer_version`, `client_version` and `bin_dir` to the DEPLOYMENT section, as it is related to "deployment" - no matter if it is deployment of installer or client...

Reply:
I am thinking about RUN section which is specific for run itself, we can run without deployment at all, on deployed env by RHHI.next for example.  OC client will be used in run itself and we will need to download it on the start of run anyway to make sure we are using version from config, or have session fixture which make sure it's installed/downloaded. the bin_dir will be probably common for whole run and we can put there more binaries files in the future so everything in this dir will be available in PATH. That's why I decided for RUN section for those.  Installer version is just for deployment, nothing else, that's why in DEPLOYMENT section.  @dahorak @vasukulkarni  are you OK with this?

Reply:
Ok